README.md

```diff
@@ -29,7 +29,9 @@ The main entry file is `pages/index.tsx` but components can be organized in `src
 Prerequisites: `node` (v10 minimum) and `npm` (v6 minimum) or `yarn` (v1.20 minimum)
```

Comment:
It doesn't seem to work on node 19. Can we give them a version here that we know will work (or use an .nvmrc) or both?

Reply:
node 19 is so new!!!

Reply:
my computer died so I was actually installing on a completely new setup haha